projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts

```diff
@@ -16,7 +16,7 @@ describe('key-value-table', () => {
 
     it('renders', async () => {
         // tslint:disable-next-line:max-line-length
-        const expectedMarkup = `<key-value-table class=\"hydrated\"><table><tbody><tr><td><button type=\"button\" id=\"undefined_undefined_0\" class=\"dot-key-value__delete__button\"><div class=\"dot-field__label\"><label for=\"undefined_undefined_0\">Delete</label></div></button></td><td></td><td></td></tr></tbody></table></key-value-table>`;
+        const expectedMarkup = `<key-value-table class=\"hydrated\"><table><tbody><tr><td><button type=\"button\" id=\"undefined_undefined_0\" class=\"dot-key-value__delete__button\"><div class=\"dot-field__label\"><label for=\"dot-undefined_undefined_0\">Delete</label></div></button></td><td></td><td></td></tr></tbody></table></key-value-table>`;
```

Comment:
`id=\"undefined_undefined_0\"` ? same for label.

Reply:
Master have that, @alfredo-dotcms? https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/dot-key-value/key-value-table/key-value-table.e2e.ts#L19

Reply:
yes, this needs work, I'll fix it